Lib/test/test_plistlib.py

```diff
@@ -838,6 +840,52 @@ def test_xml_plist_with_entity_decl(self):
                                     "XML entity declarations are not supported"):
             plistlib.loads(XML_PLIST_WITH_ENTITY, fmt=plistlib.FMT_XML)
 
+    def test_load_aware_datetime(self):
+        dt = plistlib.loads(b"<plist><date>2023-12-10T08:03:30Z</date></plist>",
+                            aware_datetime=True)
+        self.assertEqual(dt.tzinfo, datetime.UTC)
+
+    @unittest.skipUnless("America/Los_Angeles" in zoneinfo.available_timezones(),
+                         "Can't find timezone datebase")
+    def test_dump_aware_datetime(self):
+        dt = datetime.datetime(2345, 6, 7, 8, 9, 10,
+                               tzinfo=zoneinfo.ZoneInfo("America/Los_Angeles"))
+        for fmt in ALL_FORMATS:
+            s = plistlib.dumps(dt, fmt=fmt, aware_datetime=True)
+            self.assertEqual(plistlib.loads(s, fmt=fmt),
+                             datetime.datetime(2345, 6, 7, 15, 9, 10))
```

Comment:
I do not like the results of this test. It silently ignores the time zone and returns a datetime not equal to the origin. It was expected that `aware_datetime=True` is a stricter option.

Reply:
Updated to `loads` the result with `aware_datetime=True`, and compared the `loads` result with the original `dt`.

`test_dump_utc_aware_datetime` have the same issue and updated too.

Reply:
Great. Now I see that it works as expected.

But it would be nice to test also that tzinfo is UTC in the loaded datetime.

Reply:
Thanks for the review, updated.